src/components/MenuPanel/MenuPanelFooter.js

```diff
@@ -20,7 +20,12 @@ const MenuPanelFooter = ({
         </Text>
       </div>
 
-      <div css="display: inline-grid">
+      <div
+        css={`
+          display: inline-grid;
+          width: 179px;
```

Comment:
Isn’t this making the buttons always full width?

Reply:
The Preferences button yes, the label depends on the label content.

Reply:
Would it not be possible to keep the previous behavior, where both buttons use the width of the largest one? Maybe using `max-width: 100%`? Otherwise it’s something we might want to check with @dizzypaty.

Reply:
Ahhh gotcha, did not know that was the way it worked, will update this.

Reply:
By removing the fixed width the old behaviour comes back, so done.